src/processor/ulProcessor.js

```diff
@@ -1,10 +1,6 @@
 import getFlags from "../../src/services/flags.js";
 
 export function UlProcessor(config, store) {
-    
```

Comment:
Need to remove this class as it is not being used ..

Reply:
cool, I would love to :)